stripe/src/main/java/com/stripe/android/PaymentAuthWebViewStarter.kt

```diff
@@ -2,19 +2,24 @@ package com.stripe.android
 
 import androidx.activity.result.ActivityResultLauncher
 import com.stripe.android.auth.PaymentAuthWebViewContract
+import com.stripe.android.payments.StripeBrowserLauncherActivity
 import com.stripe.android.view.AuthActivityStarter
 import com.stripe.android.view.PaymentAuthWebViewActivity
 
 /**
  * A class that manages starting a [PaymentAuthWebViewActivity] with [PaymentAuthWebViewContract.Args].
  */
-internal interface PaymentAuthWebViewStarter : AuthActivityStarter<PaymentAuthWebViewContract.Args> {
+internal interface PaymentAuthWebViewStarter :
+    AuthActivityStarter<PaymentAuthWebViewContract.Args> {
     class Legacy(
         private val host: AuthActivityStarter.Host
     ) : PaymentAuthWebViewStarter {
         override fun start(args: PaymentAuthWebViewContract.Args) {
             host.startActivityForResult(
-                PaymentAuthWebViewActivity::class.java,
+                when (args.shouldUseBrowser) {
```

Comment:
This `boolean` will keep being null while `IS_BROWSER_ENABLED` is false, resulting in us sending `returnUrl == ConfirmStripeIntentParams.DEFAULT_RETURN_URL` to `PaymentAuthWebViewActivity` in some cases, will there be any issues?

Reply:
No, this is totally fine. `PaymentAuthWebViewActivity` correctly handles any variant of `returnUrl`.